Loop/View Models/BolusEntryViewModel.swift

```diff
@@ -122,7 +129,10 @@ final class BolusEntryViewModel: ObservableObject {
         // Look for Loop starting, and assume that if it started after assertCurrentPumpData that we want to call pumpManagerRefreshCompletion
         NotificationCenter.default
             .publisher(for: .LoopRunning, object: dataManager.loopManager)
```

Comment:
The "Loop Running" signal is for triggering an animation to kick off. Yes, it's associated with the pumpManagerRecommendsLoop signal, but even that isn't really a reliable signal for "assertCurrentPumpData" finished. If we want to display a spinner (and I think we don't need to for Tidepool Loop, and we don't need to add this new feature to DIY), then we probably want to make assertCurrentPumpData have an actual completion handler.